l3_dump.py

```diff
@@ -9,40 +9,198 @@
 import struct
 import subprocess
 import os
+import platform
+import shutil
+import re
 
 # ##############################################################################
 # Constants that tie the unpacking logic to L3's core structure's layout
 # ##############################################################################
 L3_LOG_HEADER_SZ = 32  # bytes; offsetof(L3_LOG, slots)
 L3_ENTRY_SZ = 32       # bytes; sizeof(L3_ENTRY)
 
-if len(sys.argv) != 3:
```

Comment:
Massive reorganization with critical improvements of the parsing logic to now use the `re` package.